NASA-AMMOS/aerie-ui
# Handle "silent" errors

Resolves #745 

Also adds null handling for all mutation requests.

To test the original error case:
1. Create a plan as a user and add some activities
2. Log out and back in as a different user and switch to the "user" role
3. Modify "utilities/permissions.ts" and make `isAdminRole` always return `true`
4. Navigate to the plan that was created in step 1
5. Attempt to edit activities
6. Verify that an appropriate error is displayed and that the UI does not freeze up anymore
